pkg/asset/ignition/bootstrap/common.go

```diff
@@ -615,6 +616,15 @@ func (a *Common) addParentFiles(dependencies asset.Parents) {
 
 		// Replace files that already exist in the slice with ones added later, otherwise append them
 		for _, file := range ignition.FilesFromAsset(rootDir, "root", 0644, asset) {
+			// We limit read access to the fencing secrets
+			match, err := regexp.MatchString(fmt.Sprintf("%s%s", ".*", fmt.Sprintf(machines.FencingCredentialsSecretFileName, "[0-9]+")), file.Path)
```

Comment:
nit: Maybe, we define a func, let's say `IsFencingCredentialsFile` in `pkg/asset/machines/master.go` to match those manifest names.

That way, we don't have to publicize `fencingCredentialsSecretFileName` constant and can just call `machine.IsFencingCredentialsFile(file.Path)`, WDYT?

Reply:
As for the function. I'm totally fine with that approach. My initial intent with the ` .*` in the regex pattern was to absorb the leading directories in the the file match check, but it does look like I can get away with it. :)

I will make the changes and confirm!

Reply:
I think I have the final changeset ready, but I'm in the process of validating this one.

Reply:
```
-rw-------. 1 root root  353 May  9 13:35 99_openshift-etcd_fencing-credentials-secrets-0.yaml
-rw-------. 1 root root  353 May  9 13:35 99_openshift-etcd_fencing-credentials-secrets-1.yaml
```
Files end up with the correct permissions on disk. :)